### DISCUSSION TITLE
Unable to Download .csv Files from DAM in Pimcore Using Public URL

### DISCUSSION THREAD

**Discussion by AlephLau** [pimcore/pimcore#17066] [Q&A / Support] [answered]
Hi everyone,

I'm currently working on a project using Pimcore, and I've encountered an issue with downloading .csv files from the Digital Asset Management (DAM) system using the public URL.

When I try to access the .csv file through its public URL, I get the following error:
No route found for "GET http://localhost:2009/route-name/name-file.csv"

Interestingly, when I use an .xlsx format, the download works correctly, but the .csv files do not. Unfortunately, I require the files to be in .csv format.

**Comment by fashxp** [comment, marked as answer]
hmm, is the csv extension missing there? https://github.com/pimcore/skeleton/blob/11.x/.docker/nginx.conf#L105

**Reply by AlephLau** [reply]
@fashxp Thanks a lot, I added it and it worked
